docs/_layouts/default.html

```diff
@@ -57,9 +57,6 @@
                 <div class="article-toc"></div>
             {% endif %}
         </div>
-
-        <!-- Concierge button anchored at the bottom of the LHN (rendered on wider screens) -->
-        {% include floating-concierge-button.html id="floating-concierge-button-lhn" %}
```

Comment:
Removal of the floating concierge button html also removed the inclusion of CONST.html. As  `CONCIERGE_CHAT_URL` was dependent on CONST.html for the main site url, this caused `Send a Message` button to fail the launch to the Concierge page.